src/client/opamSolution.ml

```diff
@@ -386,12 +386,14 @@ let parallel_apply t
       (if bypass <> !bypass_ref then
          (let spkgs = OpamSysPkg.Set.Op.(bypass -- !bypass_ref) in
           OpamConsole.note
-            "The bypass of system package %s has been registered in this switch. You \
-             can use `opam option depext-bypass-=%s' to revert."
+            "The bypass of system package%s %s has been registered in this \
+             switch. You can use `opam option depext-bypass-=%s' to revert."
```

Comment:
Possible clearer wording (for an already fairly complicated concept!):
```suggestion
            "System package%s %s %s bypassed in this \
             switch. You can use `opam option depext-bypass-=%s' to revert."
             (* ... *)
             (if OpamSysPkg.Set.cardinal skpgs > 1 then "are" else "is")
```
A further possibility might be to explain what bypassing means - i.e. describe it in terms of ignoring the depext?

Reply:
Does that still convey the idea that the change to the switch was permanently recorded ? Hm, I guess the fact that there is a way to revert it speaks for itself.

Reply:
I think it's unclear in `System package%s %s %s` but the user will see either `System package foo is` or `System packages foo and bar are` both of which imply permanence (and, as you say, so does the revert instruction!)